cookbooks/check.sh

```diff
@@ -50,19 +50,21 @@ make_lib ()
 
 echo "Checking cookbooks using $BUILD/aspect ..."
 
-(run_all_prms ) || { echo "FAILED"; exit 1; }
+( (run_all_prms ) || { echo "FAILED"; exit 1; } ) &
 
-(cd prescribed_velocity; make_lib && run_all_prms ) || { echo "FAILED"; exit 1; }
+( (cd finite_strain; make_lib && run_all_prms ) || { echo "FAILED"; exit 1; } ) &
 
-(cd free-surface-with-crust/plugin && make_lib && cd .. && run_all_prms ) || { echo "FAILED"; exit 1; }
+( (cd future && run_all_prms ) || { echo "FAILED"; exit 1; } ) &
 
-(cd future && run_all_prms ) || { echo "FAILED"; exit 1; } 
+( (cd inner_core_convection; make_lib && run_all_prms ) || { echo "FAILED"; exit 1; } ) &
 
-(cd finite_strain; make_lib && run_all_prms ) || { echo "FAILED"; exit 1; }
+( (cd prescribed_velocity; make_lib && run_all_prms ) || { echo "FAILED"; exit 1; } ) &
 
-(cd sinker-with-averaging; run_all_prms ) || { echo "FAILED"; exit 1; }
+( (cd free-surface-with-crust/plugin && make_lib && cd .. && run_all_prms ) || { echo "FAILED"; exit 1; } ) &
 
+( (cd sinker-with-averaging; run_all_prms ) || { echo "FAILED"; exit 1; } ) &
 
+wait
```

Comment:
Of course, I run this on a beefy machine (and over night). Do you suggest that I change this here?